projects/js-packages/analytics-ui/tests/jest.config.cjs

```diff
@@ -0,0 +1,7 @@
+const path = require( 'path' );
+const baseConfig = require( 'jetpack-js-tools/jest/config.base.js' );
+
+module.exports = {
```

Comment:
Do we need any of the rest of the config from https://github.com/Automattic/jetpack/blob/a7cf326619cf7280f25906a7bc8fc6c1b9347920/projects/js-packages/charts/tests/jest.config.cjs#L8?